tests/unit/backend/corpora/api_server/test_v1_collection_upload_link.py

```diff
@@ -67,7 +67,9 @@ def test__bad_link__400(self):
             test_url = furl(path=path)
             response = self.app.post(test_url.url, headers=headers, data=json.dumps(body))
             self.assertEqual(400, response.status_code)
-            self.assertEqual("The URL provided causes an error with Dropbox.", json.loads(response.data)["detail"])
+            self.assertEqual(
+                "'content-disposition' not present in the header.", json.loads(response.data)["detail"][-48:]
+            )
```

Comment:
this would fail if Dropbox switches back but oh well, we'll cross that bridge when we come to it

Reply:
nit: slightly more robust to use `assertIn()` than `["detail"][-48:]`

Reply:
this test is no longer asserting a bad link, but rather a missing header. do want to _add_ the header to maintain the original intent of the test?